crates/cxx-qt-gen/src/generator/naming/invokable.rs

```diff
@@ -23,20 +23,12 @@ impl From<&ImplItemMethod> for QInvokableName {
     fn from(method: &ImplItemMethod) -> Self {
         let ident = &method.sig.ident;
         Self {
-            name: name_from_ident(ident),
+            name: ident.clone().into(),
```

Comment:
wonder if going from `Ident` -> `CombinedIdent` like this is a good idea, as not all places may have a straight camel case conversion for the C++ side. What if we want to start prefixing all invokables or somewhere else uses this by accident then things change :thinking:

Reply:
Right, I was somewhat unsure about this myself.
Can revert.
Might especially be problematic for class names, which don't need conversion :thinking:

Reply:
I think it's better to be explicit here otherwise we could start getting lost again like the old code with one magic thing that affects unrelated places.

Reply:
Moved into associated function instead.